src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -68,6 +82,15 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
 
   public replaced;
 
+  private expandedState;
+
+  @HostBinding('@expandAnim')
+  get expandAnim() {
+    return { value: this.expandedState, params: { oldHeight: this.oldHeight } };
+  }
+
+  @HostBinding('style.height') activeHeight: string = 'auto';
```

Comment:
This should be included in the host's style just like you did with `'[style.overflow-y]': '"hidden"'`

Reply:
Yes, I think it was more dynamic when I started but now it can be moved to the basic stylesheets.

Reply:
Then again - wrong - it should be a host binding. This is only initial value.
It gets changed dynamically in onToggle()

Reply:
Yes, but this should still move to the host property of the component. The reason is that we rarely use HostBinding for style bindings.